src/runner.js

```diff
@@ -14,14 +14,12 @@ class Runner {
    * @param {string} [options.language] Options for the language of the formater
    * @memberof Runner
    */
-  constructor(options) {
+  constructor(options = { language: 'en' }) {
```

Comment:
if options is empty object, will cause option.language to be undefined too

Reply:
another way to prevent that, on ln22 pass `new MessageFormat(options.language)` to `new MessageFormat(options.language || 'en')`

Reply:
trying to get options.language when options is undefined by default will fail